nucypher/characters/lawful.py

```diff
@@ -292,16 +297,13 @@ def decrypt_message_kit(
         I/O signatures match Bob's retrieve interface.
         """
 
-        cleartexts = []
-        cleartexts.append(
-            self.verify_from(
-                data_source,
-                message_kit,
-                signature=message_kit.signature,
-                decrypt=True,
-                label=label
-            )
-        )
+        cleartexts = [self.verify_from(
+            data_source,
+            message_kit,
+            signature=message_kit.signature,
+            decrypt=True,
+            label=label
+        )]
```

Comment:
Why are we returning here a list of cleartexts, when `message_kit` always corresponds to a single message?
I know that `Bob.retrieve()` returns a list of cleartexts, but only because it's associated to `WorkOrders`, which may have more than one capsule (although currently they don't). 
This may not be an issue to solve here, but perhaps we should consider a better API for retrieve/decrypt methods: it doesn't make any sense to have methods that accept only a single capsule, but that output a list of cleartexts (of length 1). Either we make the input also a list, or we return the only cleartext.

Reply:
That's true: some hackathon participants wondered about the same thing